packages/List/src/list-item.scss

```diff
@@ -107,7 +198,7 @@
 %check {
   width: var(--ids-list-item-checkbox-check-width);
   height: var(--ids-list-item-checkbox-check-height);
-  border: .2rem solid currentcolor;
+  border: var(--ids-list-item-checkbox-check-border-width) solid currentcolor;
   border-bottom: 0;
   border-left: 0;
   transform: scaleX(-0.9) rotate(135deg);
```

Comment:
It is now alright to use rotate (https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate) this prevent having to redeclare everything if you just need to change one value of the transform shorthand.

Reply:
good to know, thanks!

Reply:
It's somehow new, IE for sure won't support this :D